### PR TITLE
Consitent store names,  adjust payment type, implement payouts

### PR DESCRIPTION
**Pull request by nanu-c**
# Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Description

This pr implements the new payout api.
* stores are now named after the same schema
* the payout page had a permission problem, when it was the first page to load
* the linter corrected a lot of things. Maybe we should activate it on all installations

image
image

# Checklist:

- [ ] I have commented my code (or ChatGPT did), particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings, neither in my IDE nor in my browser
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by lebe1 — COMMENTED** [review]
I have found two main bugs. After paying out the whole amount, the vendorID still remains on the credit list with a credit of 0,00€ as shown below. Also I'm still able to payout 0,00€.

Screenshot from 2023-10-27 14-51-13

Second bug is that I have to refresh the whole page after I clicked on the button "Auszahlen" which I could endlessly do. After refreshing the page I receive a view like the picture shown below. Do you have the same bugs as well locally?

image

File: src/views/backoffice/BackofficeVendorCreditPayout.vue [hunk 1]
```diff
@@ -17,7 +21,8 @@
             </div>
             <div>
               Für <strong>{{ vendor.LicenseID }}</strong
-              >, <strong>{{ vendor.FirstName }}</strong> <strong>{{ vendor.LastName }}</strong
+              >, <strong>{{ vendor.FirstName }}</strong>
+              <strong>{{ vendor.LastName }}</strong
               >:
```
[the remark this answers is not in the document]
Comment (nanu-c) [reply src/views/backoffice/BackofficeVendorCreditPayout.vue]: Done, thank you!

**Comment by nanu-c** [comment]
The first one is intentional. We display what the backend is delivering. Either the backend shouldn't return the Balance=0 accounts or we leave it as is. I don't want to filter the results in the frontend. But what i did now is to disable the payout button if the Balance is zero.

The second bug is solved now.

**Review by lebe1 — COMMENTED** [review]
The orders are not listed anymore in my local setup. I have tried it twice with a fresh clean up in the backend. Also is there any overflow view when the list is longer thean 10-20 orders or somethind like that?
To your comment: I think we should solve this bug in the backend then to not have vendors listed with a credit of 0€. I would put it in our TODO as a nice-to-have for now :)
image
Screenshot from 2023-10-31 16-25-34

**Comment by nanu-c** [comment]
Okey last bug solved now.

**Review by lebe1 — APPROVED** [review]

**Pull request merged**